### PR TITLE
add sublime docs

### PR DESCRIPTION
**Pull request by Bishoy-at-pieces**
ofc it needs some more polishes to be consistent as the rest of the extensions pages

### CODE REVIEW

File: docs/extensions-plugins/sublime.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+# [Pieces for Developers](https://pieces.app) | Sublime Plugin
+
+![Pieces Sublime Plugin](https://github.com/Bishoy-at-pieces/plugin_sublime/assets/167128903/272cf4cc-60ff-4bd4-8701-3e3feb38a8b1)
+
+
+##### <p align="center"> [Website](https://pieces.app/) • [Pieces OS Documentation](https://docs.pieces.app/) • [Pieces Sublime github repository](https://github.com/pieces-app/plugin_sublime)
+</p>
+
+# Important
```
Comment (shivay-at-pieces) [inline docs/extensions-plugins/sublime.mdx:17]: Can we keep the intro section similar to the rest of the extensions: 

For reference: https://github.com/pieces-app/documentation/blob/sublime-package/docs/extensions-plugins/jetbrains.mdx : 
In order to use the Pieces JetBrains Plugin, you must have [Pieces OS.](https://github.com/pieces-app/documentation/blob/sublime-package/installation-getting-started/what-am-i-installing) We recommend also using the Pieces for Developers desktop app.

:::

You can install the JetBrains plugin through the [JetBrains Marketplace](https://github.com/pieces-app/documentation/blob/sublime-package/docs/extensions-plugins/%7Bpieces_app.links.extensions.jetbrains%7D) or by searching for Pieces in the plugins search of your IDE.

Once you've installed the plugin, you can begin using Pieces without restarting your IDE.

Comment (shivay-at-pieces) [reply docs/extensions-plugins/sublime.mdx:17]: + The GIFs are not rendering in the README

Comment (shivay-at-pieces) [reply docs/extensions-plugins/sublime.mdx:17]: In general can you try to make the structure and heading similar to the rest of the extensions to maintain similarity wherever possible @Bishoy-at-pieces

Comment (Bishoy-at-pieces) [reply docs/extensions-plugins/sublime.mdx:17]: Yea same for the README on the sublime repo I think I might record another videos

File: docs/extensions-plugins/sublime.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+### Engage with the Pieces Copilot
+
+Ever wished for a personal coding assistant? Look no further! With Pieces Copilot, you can have an AI-powered assistant right at your fingertips. Simply open the command palette and select `Pieces: Copilot`. From there, you can ask anything whether it's for coding help, debugging, or even generating snippets to save to Pieces with some clicks.
+
+<Video type={'gif'} src={'https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/copilot.gif'} alt={'Copilot in Sublime'} />
+
+### Copilot Ask
+
+Need some help from the Pieces Copilot? Whether it's commenting, fixing a bug, or adding a docstring, just select the code, right-click to open the context menu, and choose `Pieces > Ask Copilot`. Let Copilot do the heavy lifting for you!
```
Comment (shivay-at-pieces) [inline docs/extensions-plugins/sublime.mdx:44]: Given the fact that we have multiple Ask methods - can you mention that you can ask about both current file and project

File: docs/extensions-plugins/sublime.mdx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+Saving a new snippet in Pieces is a breeze! Just select the code you want to save, right-click to open the context menu, and choose `Pieces > Save to Pieces`. Your code is now safely stored!
+
+<Video type={'gif'} src={'https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/create.gif'} alt={'Save to Pieces in Sublime'} />
+
+### Engage with the Pieces Copilot
+
+Ever wished for a personal coding assistant? Look no further! With Pieces Copilot, you can have an AI-powered assistant right at your fingertips. Simply open the command palette and select `Pieces: Copilot`. From there, you can ask anything whether it's for coding help, debugging, or even generating snippets to save to Pieces with some clicks.
+
+<Video type={'gif'} src={'https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/copilot.gif'} alt={'Copilot in Sublime'} />
```
Comment (shivay-at-pieces) [inline docs/extensions-plugins/sublime.mdx:40]: This is broken link @mason-at-pieces @Bishoy-at-pieces

Comment (Bishoy-at-pieces) [reply docs/extensions-plugins/sublime.mdx:40]: I think I can remove it for now(bigger space) and record new gifs and compress them better gonna check that

**Comment by mason-at-pieces** [comment]
@Bishoy-at-pieces I want to get this in but we will update it again soon after we do the product-first docs restructure. If you can just structure the current page like we have the obsidian page, that will be perfect. Like using the same exact heading and not having the community part at the end.

We will update all of this later, especially for these open source products to include sections on contributing and things like that.

**Comment by Bishoy-at-pieces** [comment]
@mason-at-pieces let me know if I should add anything else

**Comment by mason-at-pieces** [comment]
@Bishoy-at-pieces Id double-check the gifs at the bottom of the page. They are just placed back to back down there

**Review by mason-at-pieces — APPROVED** [review]
lgtm

**Pull request merged**